src/sentry/static/sentry/app/types/iconProps.ts

```diff
@@ -0,0 +1,8 @@
+export interface IconProps extends React.SVGAttributes<SVGElement> {
```

Comment:
We typically don't use `interface`s, no good reason atm, but we can easily convert this to a `type`

```jsx
type IconProps = React.SVGAttributes<SVGElement> & {
 ...
}
```

Reply:
By doing this, I get an error in the icon components that indicate iconProps.ts isn't a module.

Reply:
We also just use the `tsx` extension everywhere

Reply:
So we should make type files .tsx instead of .ts? 

![Screen Shot 2020-01-31 at 10 27 39 AM](https://user-images.githubusercontent.com/4830259/73564404-52f45380-4414-11ea-9f55-2aabee0a8a15.png)

Reply:
Note: Even if I make the change above and change it to a .tsx file I get this error: 
![Screen Shot 2020-01-31 at 10 52 34 AM](https://user-images.githubusercontent.com/4830259/73566076-d2cfed00-4417-11ea-8e9f-8f1d63dca4e4.png)

Reply:
ah sorry, did you include the `export`?